internal/logging/noop.go

```diff
@@ -0,0 +1,19 @@
+package logging
+
+type NoopLogger struct {
+}
+
+func NewNoopLogger() *NoopLogger {
```

Comment:
why not using https://pkg.go.dev/go.uber.org/zap@v1.26.0#NewNop?

Reply:
Good catch, will change it